### ISSUE TITLE
Build color variables into templates

### ISSUE THREAD

**Issue by balsama** [acquia/next-acms#30]
Tailwind comes with a large [color palette](https://tailwindcss.com/docs/customizing-colors).The provided templates use a combination of `blue-600` for links and highlights. It would be nice if this was a variable so that developers could change it in one place to customize rather than throughout the templates.

**Comment by lauriii** [comment]
Are you asking for more structure to how the colors are being selected so that all uses of one color could be changed to something else more easily? Example of this would be something like this https://tailwindcss.com/docs/customizing-colors#aliasing-color-names.

**Comment by balsama** [comment]
> Are you asking for more structure to how the colors are being selected so that all uses of one color could be changed to something else more easily?
Yes. I think I jumped to a solution in the issue summary. If I understand correctly, aliasing the current `blue-600` class to `highlight-color` or something like that would work well (given you could easily change what `highlight-color` mapped to.
